frontend/tests/features/crudCollection.test.ts

```diff
@@ -0,0 +1,42 @@
+import { TEST_ENV, TEST_URL } from "tests/common/constants";
+import { goToPage } from "tests/utils/helpers";
+import { getText } from "tests/utils/selectors";
+
+const describeIfCalledByDevEnv = TEST_ENV === "dev" ? describe : describe.skip;
+
+describeIfCalledByDevEnv("CRUD Collection", async () => {
+  it("creates and deletes a collection", async () => {
+    await goToPage(TEST_URL);
+    // LOG IN
+    await page.click(getText("Log In"));
+    await page.waitForNavigation({ url: TEST_URL, waitUntil: "networkidle" });
+    // Create collection
+    await page.click(getText("Create Collection"));
+    await page.fill("#name", "TEST COLLECTION");
+    await page.fill("#description", "TEST DESCRIPTION");
+    await page.fill("#contact-name", "TEST");
+    await page.fill("#contact-email", "TEST@example.coms");
+    await page.click("input[type=checkbox]");
```

Comment:
For robustness and readability, maybe we can add a `test-id` attribute to the checkbox and click on that, instead of relying on the form currently having only one checkbox to click? Otherwise this line won't work if the form gets a second checkbox. Also using  `test-id` attribute make this line more readable, because people will then know we're clicking on the ToS element!